src/components/mainDoctor/emptyWrapper/emptyWrapper.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { EmptyWrapper, EmptyIcon, EmptyText } from '../styles'
+
+const EmptyState = () => {
+  return (
+    <EmptyWrapper>
+      <EmptyIcon />
+      <EmptyText>
+        You have no patients yet. <br />
```

Comment:
Prevent to use BRs in you project, better solution to use styled spans. This case is fine, but when you use alot of BRs its a bad practice.

Reply:
Okey

Reply:
Usually we create two divs / spans and use it to divide text with margins